.github/workflows/validate.yml

```diff
@@ -102,14 +103,6 @@ jobs:
       uses: actions/setup-go@v4
       with:
         go-version: "${{ env.GO_VERSION }}"
-    - name: cache go mod and $GOCACHE
-      uses: actions/cache@v3
-      with:
-        path: |
-          ~/go/pkg/mod
-          ~/.cache/go-build
-        key: ${{ runner.os }}-go.sum-${{ hashFiles('**/go.sum') }}
-        restore-keys: ${{ runner.os }}-go.sum-
```

Comment:
The issue with CGO caching was resolved?

Reply:
It is irrelevant to this PR for two reasons:
1. We remove own caching here, which is sort of orthogonal to that issue
2. We only validate the dependencies here, and the issue (https://github.com/actions/setup-go/issues/368; which is not yet fixed) only surfaces when we use different distros at the same time (which we don't do here).

Reply:
(and when we use different distros at the same time and are affected, we disable caching: https://github.com/opencontainers/runc/blob/268511680f4a72b4a0595497a37e4d6a7da0215c/.github/workflows/test.yml#L70)